libraries/botbuilder-ai-luis/package.json

```diff
@@ -30,7 +30,6 @@
     }
   },
   "dependencies": {
-    "botbuilder-ai": "4.1.6"
```

Comment:
You can't do this - `botbuilder-ai-luis` depends on this package: https://github.com/microsoft/botbuilder-js/blob/main/libraries/botbuilder-ai-luis/src/index.ts#L4